resources/install.rb

```diff
@@ -48,13 +49,25 @@
   case new_resource.install_type
   when 'package'
     case node['platform_family']
-    when 'amazon', 'rhel'
+    when 'amazon'
```

Comment:
In the test we're setting both Amazon and rhel to true, so shouldn't we be doing the same actions here for Amazon and rhel?

Reply:
Is that package already available to Amazon without the uis repo?

Reply:
Sadly the UIS repo isn't available for Amazon :(